sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/ExposureNewEntryPage.java

```diff
@@ -0,0 +1,60 @@
+package org.sormas.e2etests.pages.application.contacts;
+
+import org.openqa.selenium.By;
+
+public class ExposureNewEntryPage {
+  public static final By EXPOSURE_DETAILS_KNOWN =
+      By.cssSelector("#exposureDetailsKnown span:nth-child(1)");
```

Comment:
span:nth-child(1) - if this is used, means that we can have also multiple options and we need to handle this with a single locator

Reply:
its the YES/NO/UNKNOWN buttons :
As mentioned it is not encurage on table and fields as values tend to get modified but at static buttons it does not hurt. 
There is also the xpath option: 
//div[@id='exposureDetailsKnown']//label[contains(text(), 'Yes')]
I do not mind either in this case.

Reply:
addressed